test/util.ts

```diff
@@ -377,6 +377,20 @@ describe('common/util', () => {
       stub('parseHttpRespBody', () => done()); // Will throw.
       util.handleResp(null, null, null, done);
     });
+
+    it('should surface the response body in the error message when it cannot be JSON-parsed', done => {
+      const unparseableBody = '<html>There was some error</html>';
+
+      util.handleResp(null, null, unparseableBody, err => {
+        if (err === null) {
```

Comment:
This function body could be:

```js
assert.ifError(err);
assert(err.message.includes(unparseableBody));
done();
```

Reply:
I did it this way to protect against `err` being `null`. Here's what happens otherwise:

```
test/util.ts:386:19 - error TS2531: Object is possibly 'null'.

386         assert.ok(err.message.includes(unparseableBody));
                      ~~~
```

Reply:
Woops, my suggestion didn't make sense at all given what the test is trying to do. Sorry, ignore that!